mining_test.go

```diff
@@ -45,7 +45,7 @@ func TestTxFeePrioHeap(t *testing.T) {
 		txpi, ok := prioItem.(*txPrioItem)
 		if ok {
 			if last.feePerKB < txpi.feePerKB &&
```

Comment:
Not quite.  `if txpi.feePerKB > last.feePerKB` is equivalent to `if last.feePerKB <= txpi.feePerKB`.  Notice the `<=` versus the `<` in the current code.